web/src/App.jsx

```diff
@@ -117,7 +118,7 @@ function App() {
         <div className="full-width highlighted">
           <div className="flex-stack">
             <InstallerLocaleSwitcher />
-            <InstallerKeymapSwitcher />
+            {localConnection() ? <InstallerKeymapSwitcher /> : null }
```

Comment:
This is the typical case in which using `If` is very handly: `<If condition={localConnection()} then={<InstallerKeymapSwitcher />}`